portfolio/src/main/webapp/script.js

```diff
@@ -374,4 +374,36 @@ function toggleCommentForm(loginStatus) {
 function commentsInit() {
   getComments();
   fetchLoginStatus();
+}
+
+/**
+ * Initializes the map.
+ */
+function mapInit() {
+  const homeIsolabellaCoords = { lat: 44.9067176, lng: 7.9096066 };
+  const universityCoords = { lat: 45.0623602, lng: 7.6599514 };
+
+  const map = new google.maps.Map(document.getElementById("map-container"), {
+    center: homeIsolabellaCoords,
+    zoom: 10
+  });
+
+  addLandmark(homeIsolabellaCoords, map, "H", "This is my hometown and where I live now.");
+  addLandmark(universityCoords, map, "U", 
+    "This is the Polytechnic of Turin, where I' m studying computer engineering.");
+}
+
+/**
+ * Adds a marker that shows an info window when clicked.
+ */
+function addLandmark(location, map, label, description) {
```

Comment:
Since this has a lot of arguments, consider changing the function to instead take an object parameter like this:

```function addLandmark({location, map, label, description) {}```

you can then call it like this (which makes it easier to read what the arguments are):
```
addLandmark({
  location: homeIsolabellaCoords,
  map,
  label: "H",
  description: "Some description",
});
```

Note destructuring is only available in ES6 but if you just use Chrome that should be ok.
https://ponyfoo.com/articles/es6-destructuring-in-depth